src/components/BackButton/BackButton.js

```diff
@@ -3,7 +3,7 @@ import './BackButton.css';
 
 function BackButton( { onClick } ) {
 	return (
-		<div className="back-button" onClick={ onClick }></div>
+		<div className="back-button" onClick={ onClick } />
```

Comment:
Consider removing the parenthesis and return in one line.

Reply:
Done